images/trust-manager/tests/01-version.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+if [[ "${IMAGE_NAME}" == "" ]]; then
+    echo "Must set IMAGE_NAME environment variable. Exiting."
+    exit 1
+fi
+
+set +o pipefail
+docker run --rm "${IMAGE_NAME}" 2>&1 | grep "failed to create manager"
```

Comment:
This is not an encouraging failure. Is there a way we can test that this actually comes up and works?

Reply:
I added helm tests for trust-manager